<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="seedEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="seedLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seed:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameValid">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Name is not valid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="seedValid">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Seed is not valid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeCombo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Small (128x128)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normal (256x256)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Large (512x512)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Giant (1024x1024)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unbeleivable (2048x2048)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="seedLabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QSlider" name="roughSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="seedLabel_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Roughness:</string>
   </property>
  </widget>
  <widget class="QLabel" name="roughLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
